e2e/apprunner/apprunner_suite_test.go

```diff
@@ -35,20 +37,15 @@ var _ = BeforeSuite(func() {
 	var err error
 	cli, err = client.NewCLI()
 	Expect(err).NotTo(HaveOccurred())
-	Expect(err).NotTo(HaveOccurred())
 	appName = fmt.Sprintf("e2e-apprunner-%d", time.Now().Unix())
-	err = command.Run("aws", []string{"ssm", "put-parameter", "--name", "e2e-apprunner-ssm-param", "--value", "abcd1234", "--type", "String", "--tags", "[{\"Key\":\"copilot-application\",\"Value\":\"" + appName + "\"},{\"Key\":\"copilot-environment\", \"Value\":\"" + envName + "\"}]"})
-	Expect(err).NotTo(HaveOccurred())
-	err = command.Run("aws", []string{"secretsmanager", "create-secret", "--name", "e2e-apprunner-MyTestSecret", "--secret-string", "\"{\"user\":\"diegor\",\"password\":\"EXAMPLE-PASSWORD\"}\"", "--tags", "[{\"Key\":\"copilot-application\",\"Value\":\"" + appName + "\"},{\"Key\":\"copilot-environment\", \"Value\":\"" + envName + "\"}]"})
-	Expect(err).NotTo(HaveOccurred())
+	_ = command.Run("aws", []string{"ssm", "put-parameter", "--name", ssmName, "--value", "abcd1234", "--type", "String", "--tags", "[{\"Key\":\"copilot-application\",\"Value\":\"" + appName + "\"},{\"Key\":\"copilot-environment\", \"Value\":\"" + envName + "\"}]"})
+	_ = command.Run("aws", []string{"secretsmanager", "create-secret", "--name", secretName, "--secret-string", "\"{\"user\":\"diegor\",\"password\":\"EXAMPLE-PASSWORD\"}\"", "--tags", "[{\"Key\":\"copilot-application\",\"Value\":\"" + appName + "\"},{\"Key\":\"copilot-environment\", \"Value\":\"" + envName + "\"}]"})
```

Comment:
Oh, whoops-- I didn't realize the secret discussion we had in our team mtg was about this particular e2e test 🤦🏼. In #4877 I added the `--force-overwrite-replica-secret` flag to the `create-secret` call.

Reply:
But your solution catches more cases.

Reply:
ooh initially last week it errored out because we were trying to create duplicated secrets, and then we deleted the secret manually, and it takes 7 days to delete, during which time we can't create a secret with the same name nor use the secret that's pending deletion. So at this point we are doomed and we have to change the name of the secret :rip

I don't know why it errored out initially even after the fix! Perhaps the failure happened before May 15 🤔 ?